lib/rubyflare/connect.rb

```diff
@@ -5,17 +5,30 @@ class Connect
 
     API_URL = "https://api.cloudflare.com/client/v4/"
 
-    def initialize(email, api_key)
-      @email = email
-      @api_key = api_key
+    # def initialize(email, api_key)
+    def initialize(**options)
+      if options.length == 1
+        # We are using an API token
+        @api_token = options[:api_token]
+      elsif options.length == 2
+        # Usual email and API connection
+        @email = options[:email]
+        @api_key = options[:api_key]
+      end
```

Comment:
Consider just setting them all, some will be `nil` but you're expecting it later.

```suggestion
      @api_token = options[:api_token]
      @email = options[:email]
      @api_key = options[:api_key]
```